### DISCUSSION TITLE
Fall in Void options

### DISCUSSION THREAD

**Discussion by housemazter** [BG-Software-LLC/SuperiorSkyblock2#1837] [Pending Feature Requests] [answered]
to improve admin controls and player mechanics
please add the new options:
when player fall in void?
1) keep-inventory: true|false
2) keep-exp: true|false
3) die-in-void: true|false (description: use vanilla die mode in void OR teleport player to home|spawn without dead event)
+ permissions to them. admins can use it for the donate privileges.

**Comment by Tuogptuan** [comment]
`void-teleport` in config

**Reply by Lemvs** [reply, marked as answer]
As above. For keep inventory and exp just use /gamerule.

@OmerBenGera you can mark this answered close it.
